bundles/AdminBundle/public/js/pimcore/startup.js

```diff
@@ -376,7 +376,14 @@ Ext.onReady(function () {
 
     //search element types
     pimcore.globalmanager.add("document_search_types", ["page", "snippet", "folder", "link", "hardlink", "email", "newsletter"]);
-    pimcore.globalmanager.add("asset_search_types", ["folder", "image", "text", "audio", "video", "document", "archive", "unknown"]);
+    pimcore.globalmanager.appendToArray("asset_search_types", "folder");
```

Comment:
For initial fiull we could keep the existing code, which is less, right?

Reply:
If a custom type is loaded first and then we use the add() method, we may end up overwriting the existing custom type. That's why I added the append method here.

Reply:
That's a good idea, will adapt it.

Reply:
The information is now retrieved from the backend.